tests/server.json

```diff
@@ -8,6 +8,9 @@
             "port":"9999"
         }
     },
+    "app":{
```

Comment:
While we should probably be using a matrix of versions for support, we shouldn't be making a 6 snapshot our main Preside test platform.

We should ensure this continues to work on 5 as a priority over running the main test suite in 6.

Reply:
sure, no problem, just wanted to see if it works with 6, was literally going back to mark this as a draft PR!